src/proposals/20240221/SparkEthereum_20240221.t.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import '../../SparkTestBase.sol';
+
+interface IIRM {
+    function RATE_SOURCE() external view returns (address);
+}
+
+interface IRateSource {
+    function getAPR() external view returns (int256);
+}
+
+contract SparkEthereum_20240221Test is SparkEthereumTestBase {
+
+    address public constant OLD_DAI_INTEREST_RATE_STRATEGY  = 0x512AFEDCF6696d9707dCFECD4bdc73e9902e3c6A;
+    address public constant NEW_DAI_INTEREST_RATE_STRATEGY  = 0x3C4B090b5b479402e2270C66461D6a62B2054198;
+
+    int256 public constant DAI_IRM_SPREAD =  0.016060808179122167684448000e27;
+
+    uint256 public constant WSTETH_SUPPLY_CAP_OLD = 800_000;
+    uint256 public constant WSTETH_SUPPLY_CAP_NEW = 1_200_000;
```

Comment:
I'd rename it this way, so it follows the convention
```suggestion
    uint256 public constant OLD_WSTETH_SUPPLY_CAP = 800_000;
    uint256 public constant NEW_WSTETH_SUPPLY_CAP = 1_200_000;
```

Reply:
fixed